### ISSUE TITLE
feature request : add support for some IoT protocol

### ISSUE THREAD

**Issue by avilleret** [OSSIA/libossia#397]
some of them are listed here : https://www.postscapes.com/internet-of-things-protocols/

**Comment by jcelerier** [comment]
see also 

* https://github.com/OSSIA/libossia/issues/85
* https://github.com/OSSIA/libossia/issues/83

which are maybe related a bit ?

**Comment by avilleret** [comment]
absolutely, so it makes more sense for me to dig into protocol things to understand how to implement new ones

**Comment by jcelerier** [comment]
https://github.com/redboltz/mqtt_cpp we could use that one for mqtt

**Comment by maybites** [comment]
I would like to give an upvote for mqtt.

**Comment by jcelerier** [comment]
do you have an example of an app which speaks MQTT we could use as an example to try against ?

**Comment by maybites** [comment]
not an app per se but a cool mqtt-broker with fantastic visualizations of the dataflow:

https://www.shiftr.io/desktop
